Library/Formula/v8.rb

```diff
@@ -20,6 +22,8 @@ class V8 < Formula
   depends_on :macos => :lion
 
   depends_on :python => :build # gyp doesn't run under 2.6 or lower
+  depends_on "pkg-config" => :build
+  depends_on "icu4c"
```

Comment:
Stock v8 probably vendors in an `icu4c` statically (?) rather than requiring users keep a 64MB `icu4c` installation on their system which they may not need. Node does the same thing these days.